sem_eg/multiplex.cc

```diff
@@ -37,40 +36,40 @@ class Thread {
     ::Thread* thread = static_cast<::Thread*>(arg);
     // request entry
     ThreadSemaphoreManager::Down(kSemCapacity);
-  
 
     // request access to STDOUT
-    ThreadSemaphoreManager::Down((kSemSTDOUT));
```

Comment:
Also quick question: is there a reason for the double parentheses around `kSemSTDOUT`?